controllers/vspherecluster_controller.go

```diff
@@ -97,11 +97,12 @@ func AddClusterControllerToManager(ctx *context.ControllerManagerContext, mgr ma
 		return ctrl.NewControllerManagedBy(mgr).
 			Named(controllerNameShort).
 			For(clusterControlledType).
+			WithOptions(options).
 			Watches(
 				&vmwarev1.VSphereMachine{},
 				handler.EnqueueRequestsFromMapFunc(reconciler.VSphereMachineToCluster),
 			).
-			WithOptions(controller.Options{MaxConcurrentReconciles: ctx.MaxConcurrentReconciles}).
+			WithEventFilter(predicates.ResourceNotPausedAndHasFilterLabel(ctrl.LoggerFrom(ctx), ctx.WatchFilterValue)).
```

Comment:
@sbueringer : I think filtering out paused objects via predicates should be good?!